radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
Found it:
```
    virtual void drawThumb(uint16_t x, uint16_t y, uint32_t flags) const
    {
      extern void lcdDrawBitmapPattern(int x, int y, const uint8_t * bitmap, uint32_t flags, int width=0, int offset=0);
      lcdDrawBitmapPattern(x, y, bitmap, flags);
    }
```
`lcdDrawBitmapPattern()` is defined extern but is really an inline function.